flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/adaptivebatch/AdaptiveBatchSchedulerTest.java

```diff
@@ -91,6 +96,28 @@ void setUp() {
         taskRestartExecutor = new ManuallyTriggeredScheduledExecutor();
     }
 
+    @Test
+    void testExceptionHistoryWithGlobalFailureLabels() throws Exception {
```

Comment:
for the sake of completeness, it would be nice to explicitly configure the restart strategy